recipes/ctre/build.sh

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
You probably want error checking:

```suggestion
set -euxo pipefail
```